<launch>

    <node name="portaudio_transport_publisher" pkg="portaudio_transport" type="portaudio_transport_publisher" output="screen">
        <param name="input_device" value="4" />
        <param name="frame_rate" value="" />
        <param name="frame_size" value="" />
        <param name="max_channels" value="1" />
        <param name="file_write_rate" value="10" />
    </node>

</launch>